modules/doc/content/index.md

```diff
@@ -12,27 +12,39 @@ Multiphysics Object-Oriented Simulation Environment
 !media gallery/twist_dark.mp4 style=width:100%; controls=False autoplay=True loop=True
 
 !row!
+!col! small=12 medium=4 large=4 icon=computer
+### Proven Capability class=center style=font-weight:200;
+
+!style halign=center
+
+- Scalability to over 30,000 cores
+- R&D 100 winner in 2014
+- Wide variety of [applications](application_usage/tracked_apps.md)
+
+!col-end!
+
 !col! small=12 medium=4 large=4 icon=flash_on
 ### Rapid Development class=center style=font-weight:200;
 
 !style halign=center
-Extentable [systems](syntax/index.md) for defining physics, material properties,
-postprocessing, and more.
+
+- Simple [installation](getting_started/installation/index.md)
+- Extensive [tutorials](getting_started/examples_and_tutorials/index.md)
+- Built-in [physics modules](modules/index.md)
+- User- and developer-friendly
+
 !col-end!
 
 !col! small=12 medium=4 large=4 icon=group
-### User-Focused class=center style=font-weight:200;
+### Active Community class=center style=font-weight:200;
 
 !style halign=center
-Includes [physics](modules/index.md) and [multi-scale](syntax/MultiApps/index.md) support, enabling
-collaboration.
-!col-end!
 
-!col! small=12 medium=4 large=4 icon=settings
-### Getting Started class=center style=font-weight:200;
+- Active [discussion forum](https://github.com/idaholab/moose/discussions)
+- Over 100 contributors
+- Over 500 publications
```

Comment:
Still under 600?  libMesh is up over 1600 (https://libmesh.github.io/publications.html) and I thought MOOSE was at least half of those.  Or are we only counting citations here?  IIRC the majority of people who publish a paper using libMesh don't drop us a (direct; they may cite MOOSE rather than descending into the weeds of dependencies) citation...

Reply:
This number could be out of date - I got it from some training or presentation, so maybe somebody can chime in with an updated number.

Reply:
Derek also gave the 500+ figure, so I'll leave as-is.